sdk/appconfiguration/app-configuration/tests.yml

```diff
@@ -0,0 +1,21 @@
+trigger: none
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+jobs:
+  - template: ../../../eng/pipelines/templates/jobs/archetype-sdk-integration.yml
+    parameters:
+      PackageName: "@azure/app-configuration"
+      ResourceFileLocation: sdk/appconfiguration/test-resources.json
```

Comment:
We don't use `ResourceFileLocation` anymore, it's `ResourceServiceDirectory: appconfiguration`

Reply:
Fixed!